larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -416,20 +429,24 @@ TEST_P(BConv2DOpTest, SimpleTest) {
   std::generate(std::begin(input_data), std::end(input_data), rand_generator);
   std::generate(std::begin(filters_data), std::end(filters_data),
                 rand_generator);
-  std::generate(std::begin(channel_multipliers), std::end(channel_multipliers),
-                float_generator);
-  std::generate(std::begin(bias_data), std::end(bias_data), float_generator);
+  std::generate(std::begin(post_activation_multiplier_data),
+                std::end(post_activation_multiplier_data), float_generator);
+  std::generate(std::begin(post_activation_bias_data),
+                std::end(post_activation_bias_data), float_generator);
```

Comment:
the same question about the `post_activatio_multiplier` applies here too! I guess the values can be any value not only the one listed in float_list?